ingredients.js

```diff
@@ -0,0 +1,81 @@
+const ingredientsSet = new Set()
+// ingredients = localStorage.getItem('ingredients')
+
+// ingredients.add(setIng)
+const createSpanBtns = () => {
+    const $pencilIcon = $('<i>').addClass('fa-solid fa-pencil')
+    const $trashIcon = $('<i>').addClass('fa-solid fa-trash-can')
+    
+    $trashIcon.click(event => {
+        removeIngredient(event)
+        $(event.target).parentsUntil($('.content__list')).remove()
+      })
+
+    $pencilIcon.click(event => {
+        editIngredient(event)
+    })
+    
+    return $('<span>').append($pencilIcon, $trashIcon)
+}
+
+export const getLocalStorageData = () => {
+    if (localStorage.getItem('ingredients') !== null) {
+        const receivedIngredients = localStorage.getItem('ingredients').split(',')
```

Comment:
you can move localStorage logic to the next file and abstract it away. If you do it you don't need to parse everything repeatedly